### DISCUSSION TITLE
How do I unsync files I accidentally synced into an already used directory?

### DISCUSSION THREAD

**Discussion by Polda18** [abraunegg/onedrive#3031] [Q&A] [answered]
Hi. I forgot to save my school OneDrive config and it synced my school files to the default OneDrive location, which I use for my personal OneDrive. How can I unsync the files I managed to accidentally sync there? Thank you in advance.

**Comment by abraunegg** [comment, marked as answer]
There is no facility to do this .. you will need to manually repair this.

If you have logging enabled to a file which is always recommended, you have a log of what was transferred.

**Comment by Polda18** [comment]
Is logging enabled by default?

EDIT: I'm afraid it's not enabled by default. Oh man! Do I have to open it all side by side and delete everything manually? Is there still a way to do this? I've never used Linux baremetal on my personal laptop before. I have two laptops. Gaming laptop still uses Windows 11, which I am very much satisfied with, everything works fine there, and it's very easy to use. My travel laptop used Windows 11 up until recently, and the reason I made a switch to Linux is that it was quite unstable. It ran slowly, and my Wi-Fi adapter kept crashing occassionally (especially when I left my laptop sleeping too long). I couldn't restart the Wi-Fi adapter, I had to reboot the computer to fix it. Bad if you have work in progress and cannot save it without the internet. I guess Qualcomm, who made the drivers, quite didn't make it right, because it was like this since I've actually bought the PC. Linux works much smoother, just some apps and integrations are missing here, yeah.

**Reply by abraunegg** [reply]
Only basic non-verbose logging, to system console or systemd logs .. which might not be enough for your scenario

**Reply by Polda18** [reply]
Oh boy. I guess I'll have to sit and manually delete all unwanted files from my personal OneDrive directory one by one via comparison method. That's gonna be quite mundane Saturday I guess.
